example/README.md

```diff
@@ -11,6 +11,8 @@ This directory contains examples for the usage of the `toml.dart` package.
  - [`./toml_encoder`][toml-dart/example/toml_encoder] contains an example for how to use the `TomlEncoder` from the `toml.encoder` library to encode a `Map` as a TOML document.
 
  - [`./toml_parser`][toml-dart/example/toml_parser] contains an example for how to use the `TomlParser` from the `toml.decoder` library to decode a TOML document from a string.
+ 
+ - [`./flutter_example`][toml-dart/example/flutter_example] contains a Flutter example for how to use the `toml.dart` library to decode a TOML document as an asset string.
```

Comment:
I'd try to keep this list in alphabetical order. Same for the link reference definition below.